platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationImpl.java

```diff
@@ -149,7 +150,7 @@ public <T> List<T> getValues(
             @Nullable final List<T> defaultValue) {
         ArgumentUtils.throwArgBlank(propertyName, "propertyName");
         Objects.requireNonNull(propertyType, "propertyType must not be null");
-        if (!exists(propertyName)) {
+        if (!exists(propertyName) || propertyType.equals(CloudBucketConfig.class)) {
```

Comment:
Why is this change required?

Reply:
It won't be needed the changes in config implementation are present (see [this](https://github.com/hashgraph/hedera-services/pull/17010#discussion_r1891669593))